README.rst

```diff
@@ -19,7 +19,7 @@ You can install the auth0 Python SDK using the following command.
 For python3, use the following command
 
 .. code-block:: python
-    
+
```

Comment:
<img width="260" alt="image" src="https://user-images.githubusercontent.com/1299658/154289851-30de6a29-e33e-4892-92d1-74361eda2f6d.png">

My editor decided to remove a bunch of whitespace